packages/block-library/src/columns/style.scss

```diff
@@ -10,41 +10,36 @@
 }
 
 .wp-block-column {
-	flex: 1;
+	flex-grow: 1;
 	margin-bottom: 1em;
 
 	// Responsiveness: Show at most one columns on mobile.
 	flex-basis: 100%;
 
-	// Beyond mobile, allow 2 columns.
-	@include break-small() {
-		flex-basis: 50%;
-		flex-grow: 0;
-	}
-
 	// Prevent the columns from growing wider than their distributed sizes.
 	min-width: 0;
 
 	// Prevent long unbroken words from overflowing.
 	word-break: break-word; // For back-compat.
 	overflow-wrap: break-word; // New standard.
 
-	// Add space between columns. Themes can customize this if they wish to work differently.
-	// Only apply this beyond the mobile breakpoint, as there's only a single column on mobile.
 	@include break-small() {
-		&:nth-child(odd) {
-			margin-right: $grid-size-large * 2;
-		}
-		&:nth-child(even) {
-			margin-left: $grid-size-large * 2;
-		}
 
-		&:not(:first-child) {
-			margin-left: $grid-size-large * 2;
+		// Beyond mobile, allow 2 columns.
+		flex-basis: 50%;
```

Comment:
If we're getting rid of the 2-column breakpoint, then `flex-basis:auto;` would make more sense here wouldn't it?